### PR TITLE
Mobile gnav breadcrumbs new design

### PR DESCRIPTION
**Pull request by bandana147**
* Added breadcrumbs to sections
* Hidden breadcrumbs on first screen of sidebar

Resolves: [MWPW-161558](https://jira.corp.adobe.com/browse/MWPW-161558)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mobile-gnav-breadcrumbs--milo--adobecom.hlx.page/?martech=off

QA: https://main--federal--adobecom.hlx.page/federal/dev/blaishram/page-with-localnav?milolibs=mobile-gnav-breadcrumbs&newNav=true

### CODE REVIEW

**Review by salonijain3 — APPROVED** [review]

File: libs/blocks/global-navigation/utilities/utilities.js [hunk 1]
```diff
@@ -442,7 +443,7 @@ export const transformTemplateToMobile = (popup, item, localnav = false) => {
       </span>
     </div>
     <div class="title">
-      <div class="breadcrumbs"></div>
+      ${breadCrumbs}
```
Comment (sharmrj) [inline libs/blocks/global-navigation/utilities/utilities.js:446]: do we want to do `${isHomePage() : '' : breadcrumbs}`?

**Review by sharmrj — APPROVED** [review]

**Review by sharmrj — COMMENTED** [review]
@bandana147 there seems to be an issue with your branch for example on
https://main--homepage--adobecom.hlx.live/homepage/index-loggedout?milolibs=mobile-gnav-breadcrumbs&newNav=true
Screenshot 2024-11-07 at 11 27 29 AM

File: libs/blocks/global-navigation/utilities/utilities.js (hunk 1 above)
Comment (bandana147) [reply libs/blocks/global-navigation/utilities/utilities.js:446]: We are not showing breadcrumbs in homepage

**Comment by sharmrj** [comment]
Screenshot 2024-11-07 at 11 56 44 AM
This happens too. The caret isn't on the far end.

File: libs/blocks/global-navigation/utilities/utilities.js (hunk 1 above)
Comment (sharmrj) [reply libs/blocks/global-navigation/utilities/utilities.js:446]: Yes but we need that empty breadcrumbs div to make sure we have the right amount of space above the mega menu title.

**Comment by bandana147** [comment]
> @bandana147 there seems to be an issue with your branch for example on https://main--homepage--adobecom.hlx.live/homepage/index-loggedout?milolibs=mobile-gnav-breadcrumbs&newNav=true Screenshot 2024-11-07 at 11 27 29 AM

This is fixed.

**Comment by bandana147** [comment]
> Screenshot 2024-11-07 at 11 56 44 AM This happens too. The caret isn't on the far end.

Fixed!

**Pull request merged**